lib/commanded/process_managers/process_manager_instance.ex

```diff
@@ -350,6 +348,33 @@ defmodule Commanded.ProcessManagers.ProcessManagerInstance do
     end
   end
 
+  defp handle_after_command([], %State{} = state) do
+    %State{idle_timeout: idle_timeout} = state
+
+    {:noreply, state, idle_timeout}
+  end
+
+  defp handle_after_command([command | commands], %State{} = state) do
+    %State{
+      process_manager_module: process_manager_module,
+      process_state: process_state
+    } = state
+
+    case process_manager_module.after_command(process_state, command) do
+      :stop ->
+        Logger.debug(fn ->
+          describe(state) <> " has been stopped by command " <> inspect(command)
+        end)
+
+        :ok = delete_state(state)
```

Comment:
I don't know if it's correct to delete instance here, in the `c:interested?` it's done by `ProcessRouter` but in this case i don't see any way to handle by it

Reply:
Doing this is fine as the process router calls the process manager instance to delete its state.

Reply:
Ohh thank you 🙏🏿